package.json

```diff
@@ -3,18 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "node server.js",
```

Comment:
I think we need `nodemon` here instead since now we lost Next.js' live reloading feature :(

Reply:
Nope, I take that back - we don't need nodemon 😬